src/style.css

```diff
@@ -0,0 +1,126 @@
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  --color-blue: #00acdc;
```

Comment:
Better to add variables to the root selector

[More info.](https://developer.mozilla.org/ru/docs/Web/CSS/:root)